models/passenger.js

```diff
@@ -0,0 +1,101 @@
+const { db } = require('../database/connection.js')
+
+const getPassengerIdByName = (name) => {
+  if(typeof name !== 'string') {
+    return 'Please enter a passenger name.'
+  }
+  return db.one('SELECT id FROM passengers WHERE name = $1', [name])
+    .then( response => {
+      return response
```

Comment:
this can be one line:    .then( response => response )
you can leave out the curlys and return